frontend/packages/gitops-plugin/src/components/details/CommitDetails.tsx

```diff
@@ -80,11 +84,34 @@ const CommitDetails: React.FC<CommitDetailsProps> = ({ imageName }) => {
     <>
       {commitData.id ? (
         <>
-          <Timestamp timestamp={commitData.timestamp} />
-          {t('gitops-plugin~{{id}} by {{author}}', {
-            id: commitData.id,
-            author: commitData.author,
-          })}
+          <Split className="odc-gitops-commit">
+            <SplitItem isFilled>
+              <Label className="odc-gitops-commit__item" isTruncated>
+                <Timestamp timestamp={commitData.timestamp} />
+              </Label>
+            </SplitItem>
+          </Split>
+          <Split className="odc-gitops-commit">
+            <SplitItem isFilled>
+              <Label className="odc-gitops-commit__item" isTruncated>
+                {t('gitops-plugin~{{id}}', { id: commitData.id })}
```

Comment:
If you only pass the msg it is not required to use i18n here. IMHO.

Reply:
This has been updated.